mobile/src/main/res/values-de/strings.xml

```diff
@@ -161,27 +163,43 @@
     <string name="info_write_tag_unsupported">Dieses Gerät unterstützt kein NFC</string>
     <string name="info_write_tag_disabled">NFC ist deaktiviert</string>
     <string name="info_openhab_apiversion_label">openHAB-Rest-API-Version</string>
-    <string name="info_openhab_uuid_label">openHAB-UUID</string>
     <string name="info_openhab_push_notification_label">Push-Benachrichtigungsstatus</string>
     <string name="action_settings">Einstellungen</string>
     <string name="nfc_info_hint">Item-Updates können auf NFC-Tags geschrieben werden. Sie können hier ein Item auswählen oder ein Widget auf der Sitemap anklicken.</string>
     <string name="info_not_set">Nicht gesetzt</string>
     <string name="empty_page">Diese Seite beinhaltet keine sichtbaren Widgets</string>
     <!-- Background tasks -->
     <string name="send_device_info_to_server">Geräteinformationen an Server senden</string>
+    <string name="send_device_info_to_server_short">Geräteinformationen senden</string>
+    <string name="send_device_info_category_event_based_title">Ereignisbasiert</string>
+    <string name="send_device_info_category_event_based_explanation">Diese Items werden bei entsprechenden Ereignissen aktualisiert, z.B. bei einem eingehenden Anruf</string>
+    <string name="send_device_info_category_schedule_based_title">Basierend auf einem Zeitplan</string>
+    <string name="send_device_info_category_general">Allgemein</string>
     <string name="send_device_info_item_prefix">Präfix des Item-Namens</string>
     <string name="send_device_info_item_prefix_summary">Die Item-Namen der folgenden Funktionen haben dieses Präfix \'%s\'</string>
     <string name="send_device_info_item_prefix_summary_not_set">Für die Item-Namen der folgenden Funktionen ist kein Präfix gesetzt</string>
     <string name="settings_item_update_pref_howto_summary">Hier tippen, um Hinweise zu erhalten, wie man dieses Feature serverseitig einrichtet</string>
+    <string name="settings_alarm_clock">Weckzeit</string>
     <string name="settings_alarm_clock_summary_on">Sendet Weckzeit an Item \'%1$s\'</string>
-    <string name="settings_phone_state_summary_on">Sendet Anrufstatus zu Item \'%1$s\'</string>
+    <string name="settings_alarm_clock_summary_off">Weckzeit nicht senden</string>
+    <string name="settings_phone_state">Anrufstatus</string>
+    <string name="settings_phone_state_summary_on">Sendet Anrufstatus an Item \'%1$s\'</string>
```

Comment:
@mueller-ma I wonder whether we shouldn't make those consistent to the off variants? `Anrufstatus an Item \'%1$s\' senden` or even `Anrufstatus wird an Item ... gesendet`?

Reply:
`Anrufstatus wird an Item ... gesendet` sound better to me. I updated all `summary_on` strings.